SampleCode/Python/sample_identify_cross_page_tables.ipynb

```diff
@@ -4,17 +4,28 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "# Identify cross-page tables based on rules\n",
+    "# Identify cross-page tables and merge them into one table based on rules\n",
     "\n",
```

Comment:
merge them into one table in the markdown output

Reply:
fixed